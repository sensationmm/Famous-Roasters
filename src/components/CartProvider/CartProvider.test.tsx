import { render, waitFor } from '@testing-library/react'
import React from 'react'

import { CartProvider } from '.'

describe('Cart provider component', () => {
  it('Renders correctly', async () => {
    const { container } = render(
      <CartProvider>
        <span />
      </CartProvider>,
    )
    await waitFor(() => new Promise((res) => setTimeout(res, 0)))
    expect(container).toMatchSnapshot()
  })
})
