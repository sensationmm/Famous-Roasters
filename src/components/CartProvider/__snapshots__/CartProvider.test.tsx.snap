// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart provider component Renders correctly 1`] = `
<div>
  <span />
</div>
`;
